### PR TITLE
Changed the Helm version and removed BETA

### PR DESCRIPTION
**Pull request by tdensmore**
I would appreciate a technical review here - but as far as I know we are using Helm v3 be default, and it is no longer BETA.

### CODE REVIEW

**Review by samantha-replicated — DISMISSED** [review]
LGTM! I'll defer to @morningvera to confirm technical accuracy before merging.

**Review by morningvera — APPROVED** [review]

**Review by morningvera — DISMISSED** [review]
just realized you changed the default. helm v2 has always been the default and changing it to v3 would be a breaking change so i suggest not editing that part of the docs.

**Comment by JustinN-Replicated** [comment]
Agree with @morningvera that the 'default' we use hasn't been changed at all (at least for replicated helm install). Alluded to this in another file now also added to the pr.

File: content/kots-cli/pull.md
Comment (samantha-replicated) [inline content/kots-cli/pull.md:26]:
```suggestion
| `--helm-version` | string | the Helm version with which to render the Helm Chart _(default `"v2"`)_. |
```

**Comment by samantha-replicated** [comment]
Ok, I just committed a change to ensure we were clear V2 is the default. Otherwise, it seems like we have ironed out everything in this PR? @tdensmore do you need one of us to merge for you?

**Comment by ajp-io** [comment]
@JustinN-Replicated @tdensmore Can this be closed, merged, or updated and then merged?

**Comment by JustinN-Replicated** [comment]
Not too sure why this wasn't merged back in December. @replicatedhq/replicated-docs  in case you wanted to take a look. I made some minor changes. @tdensmore if you could review that would be helpful too. Aside from that, lgtm

**Comment by tdensmore** [comment]
Looks good to me. Can I ask why the default Helm install method is still `v2`? Helm 2 should never be used AFAIK...

**Comment by JustinN-Replicated** [comment]
from Vera's comment above: https://github.com/replicatedhq/kots.io/pull/617#pullrequestreview-818144145

**Review by JMNordeste-dev — APPROVED** [review]

**Comment by jonquil2002** [comment]
@JustinN-Replicated Please lmk when we can close this and migrate the content diff to the new website. Trying to close out all of the PRs in this repo now. Thanks!

**Review by JMNordeste-dev — APPROVED** [review]

**Review by JMNordeste-dev — APPROVED** [review]

**Comment by JustinN-Replicated** [comment]
@jonquil2002 all set

**Pull request merged**
